File: gitlfs-client/src/main/java/ru/bozaro/gitlfs/client/exceptions/UnauthorizedException.java
package ru.bozaro.gitlfs.client.exceptions;

import org.apache.http.HttpResponse;
import org.apache.http.client.methods.HttpUriRequest;

import javax.annotation.Nonnull;

/**
 * Unauthorized HTTP exception.
 *
 * @author Artem V. Navrotskiy
 */
public class UnauthorizedException extends RequestException {
  public UnauthorizedException(@Nonnull HttpUriRequest request, @Nonnull HttpResponse response) {
    super(request, response);
  }
}

File: gitlfs-client/src/main/kotlin/ru/bozaro/gitlfs/client/exceptions/UnauthorizedException.kt
package ru.bozaro.gitlfs.client.exceptions

import org.apache.http.HttpResponse
import org.apache.http.client.methods.HttpUriRequest

/**
 * Unauthorized HTTP exception.
 *
 * @author Artem V. Navrotskiy
 */
class UnauthorizedException(request: HttpUriRequest, response: HttpResponse) : RequestException(request, response)
